packages/terra-functional-testing/package.json

```diff
@@ -14,7 +14,7 @@
     "*.md"
   ],
   "scripts": {
-    "compile": "babel src --out-dir lib --copy-files",
+    "compile": "babel src --out-dir lib --copy-files --ignore src/terra-cli",
```

Comment:
Won't this need to be in lib once terra-functional-testing is released?

Reply:
it still gets moved, just not transpiled.